app/main.py

```diff
@@ -1,3 +1,47 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: Distance | int | float) -> Distance:
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+        else:
+            self.km = self.km + other
+
+        return self
+
+    def __mul__(self, other: int | float) -> Distance:
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: int) -> Distance:
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: int | float) -> bool:
+        return self.km < other
+
+    def __le__(self, other: int | float) -> bool:
+        return self.km <= other
+
+    def __gt__(self, other: int | float) -> bool:
+        return self.km > other
+
+    def __ge__(self, other: int | float) -> bool:
+        return self.km >= other
+
+    def __eq__(self, other: int | float) -> bool:
+        return self.km == other
```

Comment:
u didn't fix that, other also could be an instance of Disctance class in this methods
![image](https://github.com/mate-academy/py-distance-class/assets/103201010/c166a734-fa7f-428e-af29-d76126a1e1a6)